### ISSUE TITLE
I have 'Periodically check for update' disabled but i still get update notifications

### ISSUE THREAD

**Issue by jagflashardinxbox360** [AdAway/AdAway#2260]
**Describe the bug**
Hello!
I've disabled update check of AdAway but i still get notifications about it in the app.

**Expected behavior**
I expect that it won't do an update check when i have it disabled.

**Screenshots**
![image](https://user-images.githubusercontent.com/12037193/96274516-dcfb4100-0fd0-11eb-9042-2aed65306037.png)

**Device**
 - AdAway version: 5.0.9 (i can't test version 5.0.10 until you release the next version)
 - Device: Samsung S9+
 - Android version: Android 10
 - Root version Magisk 20.4

**Additional context**
I've now updated to 5.0.10 so i don't know if this has been fixed yet but it didn't worked in 5.0.9.

**Comment by PerfectSlayer** [comment]
Hello @jagflashardinxbox360 

Thanks for the report.
What kind of notification do you have? Android notification? In app update notification?
About the update, is it about the app itself or the hosts files?

**Comment by jagflashardinxbox360** [comment]
It's the notification i get in the upper right corner of AdAway. I can read `Update available`, so it's the app itself.

**Comment by PerfectSlayer** [comment]
I get it, there is no "Check for (application) update at start up".
The application won't look for (application) update in background but will check it when you start it.

It is really an issue? You will be missing application updates whereas I try to inform users to not miss it.
As the application is not in Google Play Store, it's the only way I found. Moreover, there is no Android notification to not bother the users.

**Comment by jagflashardinxbox360** [comment]
It seems that it is now possible to disable check for updates at startup and that's really great, but now there's no button to check for updates manually and the last changes text will also disappear after i have updated the app. I don't know how it should behave but wouldn't it be great to leave the Last changes?

**Comment by PerfectSlayer** [comment]
You can manually check for update using the refresh/check button.
You can also see the last changes by taping the app version.

See picture for details:

![image](https://user-images.githubusercontent.com/1766222/111027967-d17f3b00-83f3-11eb-9eb5-fbf45a748eb4.png)

**Comment by jagflashardinxbox360** [comment]
No, that doesn't work. If i press the app version i only get a red screen with no text.

![image](https://user-images.githubusercontent.com/12037193/111033152-1add8380-8410-11eb-913a-a45f91d808f1.png)

The refresh/check button will only check the hosts sources and not the app. At least i can't see it do an app check when i press it.

**Comment by PerfectSlayer** [comment]
Hum, that is an issue! 

The app will check for update when it display its main screen.
If no update is found, it will display the current version number. Tapping it should display the last changelog of your current version.
If an update is found, it will display "update available". Tapping it should display the changelog of the new version and a download button.

Did you update the app since you reported the bug? If so, which version are you using? (and XDA/GitHub or F-Droid?)

**Comment by Sopor** [comment]
I'm using the AdAway-5.4.0-210228.apk downloaded from GitHub.

**Comment by Sopor** [comment]
I'm using the AdAway-5.4.0-210228.apk downloaded from GitHub and i have the same issue.

**Comment by PerfectSlayer** [comment]
From you device, can you try to load the following URL: http://app.adaway.org/manifest.json

If you successfully load it, can provide a logcat of the issue?
Please enable the verbose logging from the app, restart the app and try to load the changelog.
You should see some errors in the log about loading manifest.

**Comment by jagflashardinxbox360** [comment]
Sorry for my late reply, but I've been away for a couple of days.
My Adaway version is 5.4.0 and I've downloaded it from github.
I'm not sure where you want me to run the manifest.json file, please explain.

I'm not sure what you mean about a logcat, but do you want me to use an external app to get the logcat or do you want me to send you something from Adaway? This is a logcat reader, can I use that (https://play.google.com/store/apps/details?id=com.dp.logcatapp)? What I understand the logcats can be quite large so what do you want me to log?
